.circleci/config.yml

```diff
@@ -230,8 +230,8 @@ jobs:
           command: ./gradlew assembleSelfSignedRelease
 
       - run:
-          name: Check APK size hasn't increased by more than 100kb
-          command: if [ $(ls -l collect_app/build/outputs/apk/selfSignedRelease/*.apk | awk '{print $5}') -gt 10734563 ]; then exit 1; fi
+          name: Check APK size hasn't increased too much
+          command: if [ $(ls -l collect_app/build/outputs/apk/selfSignedRelease/*.apk | awk '{print $5}') -gt 10800000 ]; then exit 1; fi
```

Comment:
@lognaturel @seadowg I had to increase the limit here. The difference is not big so I guess it's ok but wanted to let you know.

Reply:
Yeah this seems fine. I like this process of us getting a warning so we know it's creeping up. I'll add a note to the Google Drive stuff to make sure we drop this again.